microservices/recordings/src/Service/Encoder.php

```diff
@@ -344,6 +345,12 @@ private function getAccCdrs(RawRecordingInfo $rawRecordingInfo): array
 
     private function isProcessableFile(RawRecordingInfo $rawRecordingInfo): bool
     {
+        try {
+            return $this->recordingEndedChecker->execute($rawRecordingInfo->getFullName());
+        } catch (\Exception $e) {
+            // noop
+        }
+
```

Comment:
The true or false value returned by this service is not used anywhere.

I would see more reasonable to add another boolean check here like the others in this method ($isTooRecent, $isTooSmall), maybe $isStillRecording?